intranet/static/js/signage.js

```diff
@@ -19,4 +21,4 @@ $(document).ready(function() {
             $('html').css('cursor', 'auto');
         }
     });
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2746716151/issues/source?bid=3176160&fileBranchId=3355598#l24)